### PR TITLE
Updated readme.md

### PR DESCRIPTION
**Pull request by nehaalhat**
1. Updated readme.md
2. Removed configuration.md.

### CODE REVIEW

**Review by maelk — COMMENTED** [review]
does this PR miss the dev-setupd.md docs that was moved out of the Readme ?

**Comment by nehaalhat** [comment]
We have already raised PR for dev-setup.md under docs and it is merged, so removed those  steps from readme.md and added dev-setup.md reference under resources.

**Comment by digambar15** [comment]
Looks good!

File: README.md [hunk 1]
```diff
@@ -1,76 +1,25 @@
 # Hardware classification controller
```
Comment (digambar15) [inline README.md:2]: I would like to mention the goal of HWCC, what you mentioned below is more of implementation details. First elaborate what's goal of the project then you can keep below "how" part as is.

**Review by digambar15 — CHANGES_REQUESTED** [review]
Please work on comments

**Review by digambar15 — APPROVED** [review]
Looks good

**Pull request merged**